### ISSUE TITLE
[T3] [Accessibility] [Functional] unable to navigate from parent tab to Add

### ISSUE THREAD

**Issue by naveennayakp** [Mohanme13/Feb-19-Bug-bash#162]
unable to navigate from Problems tab to Add section using tab
